team/static/customview.js

```diff
@@ -0,0 +1,333 @@
+var trace_3d, layout_3d;
+var updates_3d = []
+var groups_3d = []
+
```

Comment:
These methods are all camel cased.  Change to underscore case for consistency and adherence to coding standard

Reply:
The methods/functions are in camel case which is according to the coding standard specified in the google doc. 

Specification in Google doc
----------------------------
Functions
Function names are written in camelCase
Line up brace with function
e.g. 
function getAverage(x, y) {
	return ((x + y) / 2);
}

Reply:
That is the javascript section.  The python function says underscore case.  And regardless, there are some functions in other files that have underscore case, so this is inconsistent regardless.  One set needs to be fixed.  Going off of the drive document, however, look at page 4.

Reply:
This is resolved. There was some confusion regarding the code being reviewed.